packages/provider/src/wallet.ts

```diff
@@ -728,6 +732,20 @@ export const useWallet = () => {
   return walletInstance
 }
 
-export const resetWallet = () => {
-  walletInstance = undefined
+export const connectWallet = async (network?: string | number, options?: ConnectOptions & { walletConfig?: Partial<ProviderConfig> }) => {
+  if (!walletInstance) {
+    walletInstance = getWallet(network, options?.walletConfig)
+  }
+  if (network && options) {
+    options.networkId = network
```

Comment:
passing network here isn't the same as setting the default network ID in configWallet or getWallet - just as a note.
also don't see an advantage for having this over just making people call both get and connect methods.

Reply:
I think its necessary for expected use, and shouldn't break things.. cuz people may want to call connectWallet() multiple times will using the same instance